webproject-frontend/src/vuex/index.js

```diff
@@ -19,16 +19,68 @@
  */
 import Vue from "vue";
 import Vuex from "vuex";
+import axios from "axios";
 
 Vue.use(Vuex);
 
 export default new Vuex.Store({
   state: {
-    count: 0
+    initialized: false,
+    currentUser: null,
+    theme: null
+  },
+  actions: {
+    loadCurrentUser({ commit }) {
+      axios
+        .get("/api/users/current")
+        .then(response => {
+          commit("setCurrentUser", response.data);
+          commit("initialized");
+        })
+        .catch(error => {
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/gmasil/Webproject-NG/pullRequest?prid=8166316)
```suggestion
        .catch((error) => {
```